src/CSET/operators/plot.py

```diff
@@ -411,29 +411,38 @@ def _plot_and_save_postage_stamp_spatial_plot(
 
 
 def _plot_and_save_line_series(
-    cube: iris.cube.Cube, coord: iris.coords.Coord, filename: str, title: str, **kwargs
+    cube: iris.cube.Cube | iris.cube.CubeList,
+    coord: iris.coords.Coord,
+    filename: str,
+    title: str,
+    **kwargs,
 ):
     """Plot and save a 1D line series.
 
     Parameters
     ----------
-    cube: Cube
-        1 dimensional Cube of the data to plot on y-axis.
+    cube: Cube or CubeList
+        Cube or CubeList containing the cubes to plot on the y-axis.
     coord: Coord
-        Coordinate to plot on x-axis.
+        Coordinate to plot on the x-axis.
     filename: str
         Filename of the plot to write.
     title: str
         Plot title.
     """
     fig = plt.figure(figsize=(10, 10), facecolor="w", edgecolor="k")
-    iplt.plot(coord, cube, "o-")
+
+    for cube_iter in cube:
```

Comment:
```suggestion
    for cube_iter in iter_maybe(cube):
```
As this could be a non-iterable cube, we should ensure it is iterable first.